blockchain/genesis.go

```diff
@@ -364,33 +364,28 @@ func GenesisBlockForTesting(db database.DBManager, addr common.Address, balance
 	return g.MustCommit(db)
 }
 
-// DefaultGenesisBlock returns the Klaytn main net genesis block.
+// DefaultGenesisBlock returns the Cypress mainnet genesis block.
+// It is also used for default genesis block.
 func DefaultGenesisBlock() *Genesis {
```

Comment:
```suggestion
func DefaultCypressBlock() *Genesis {
```

Reply:
As mentioned in the comment, it is also used for default genesis block. So, I didn't changed the name.